### ISSUE TITLE
[quest] Defend Stoneclaw Castle! - quest log indicator

### ISSUE THREAD

**Issue by SharpiesOfWar** [WarEmu/WarBugs#2532]
Hello.

I found that when the quest is completed (20/20 kills), the completion status does not show up as completed in the quest log entry. It does however show up as completed when you have the specific quest opened.

**Comment by Evildoor** [comment]
Have you got this in one play session? Or you got 20 kills, logged out, logged in and then checked?

**Comment by SharpiesOfWar** [comment]
Hi.

At that time, I had finished the quest in that play session. I also had another "Defend..." quest and it did show as completed properly in both the quest log and when you open the quest. It was a 25/25 kills one. 
I also just logged into another character just now that has the quest "Defend Stoneclaw Castle" and in the quest log, it shows that it is not complete, but opening the quest it shows it is completed, so I have the latter case you mentioned as well. Looking across my chars, I see this on a number of characters for specific "Defend...xxx" quests. Some are being updated properly while others are not. I will open issues on those ones too and include the test to see if it remains if I log out and log back in as well.

**Comment by Evildoor** [comment]
No need for test. AFAIK, all quests are affected by this bug if you log off and on again, so i wanted to understand whether it was the case here or not. However, you say that you had this problem even without logging off, which is something new.

**Comment by SharpiesOfWar** [comment]
Hi. I just completed some quests and they did get marked as completed, then did the log in and log out and found they were no longer marked completed in the log as you described in your reply. 

I almost always turn in my kill quests at the same time once they are all done, which is what I was using to base my reply on it not being marked completed in my quest log. However, knowing now that logging out and back in wipes the completion status of quests in the quest log, it might have been possible that it was completed prior and I forgot to turn it in.

I had 3 kill quests in my quest log at the time. One with not enough kills, marked correctly, the quest I indicated, which had 20/20 that was not marked, and the 25/25 that was marked completed correctly. Maybe close this issue until I start with a clean slate of quests and can confirm for sure. Sorry.
